src/main/java/de/utkast/sputnik/SputnikTest.java

```diff
@@ -0,0 +1,19 @@
+package de.utkast.sputnik;
+
+public class SputnikTest {
+
+	// findbugs warning
+	public boolean wrongStringComparsion(String a, String b) {
+		return a == b;
```

Comment:
\[FindBugs\] WARNING: ES: Comparison of String parameter using == or \!= in de\.utkast\.sputnik\.SputnikTest\.wrongStringComparsion\(String, String\)